### ISSUE TITLE
[Support]: Frigate will not start after updating to HAOS 11.5 

### ISSUE THREAD

**Issue by Ralphy140** [blakeblackshear/frigate#9741]
### Describe the problem you are having

Frigate was fully configured and working including now will not start after upgrading HAOS to 11.5 

### Version

0.13.01

### Frigate config file

```yaml
ffmpeg:
  hwaccel_args: preset-vaapi

mqtt:
    host: 172.16.254.25
    user: *
    password: *

detectors:
  coral:
    type: edgetpu
    device: usb
go2rtc:
  streams:
    pool_sub: 
      -  "ffmpeg:http://172.16.254.40/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=admin&password=*"
    pool_main: 
      - "ffmpeg:http://172.16.254.40/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=admin&password=*#video=copy#audio=copy#audio=opus"

    #- rtsp://admin:*@172.16.254.40:554/h264Preview_01_main
    # - "ffmpeg:rtsp_cam_sub#audio=opus"
    backdeck_main: 
      - "ffmpeg:http://172.16.254.41/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=admin&password=*#video=copy#audio=copy#audio=opus"
      #- rtsp://admin:*@172.16.254.41:554/h265Preview_01_main
      #- "ffmpeg:rtsp_cam_sub#audio=opus"
    backdeck_sub: 
      - "ffmpeg:http://172.16.254.41/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=admin&password=*"
    frontyard_sub: 
      - "ffmpeg:http://172.16.254.42/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=admin&password=*#video=copy#audio=copy#audio=opus"
    frontyard_main: 
      - "ffmpeg:http://172.16.254.42/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=admin&password=*#video=copy#audio=copy#audio=opus"
      #- rtsp://admin:*@172.16.254.42:554/h265Preview_01_main
      #- "ffmpeg:rtsp_cam_sub#audio=opus"
    Doorbell_main: 
      - "ffmpeg:http://172.16.254.220/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=admin&password=*#video=copy#audio=copy#audio=opus"
    Doorbell_sub: 
      - "ffmpeg:http://172.16.254.220/flv?port=1935&app=bcs&stream=channel0_sub.bcs&user=admin&password=*"



cameras:
  doorbell: # <------ Name the camera
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/Doorbell_main?video=copy&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://127.0.0.1:8554/Doorbell_sub?video=copy&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - detect

    detect:
      fps: 5
      enabled: True
    record:
      enabled: True
      retain:
        days: 1
        mode: motion
      events:
        retain:
          default: 1
    snapshots: # <----- Enable snapshots
      enabled: True
      bounding_box: true
      retain:
        default: 15
    objects:
      track:
      - person
      - dog
      - cat
    live:
      stream_name: Doorbell_main
  frontyard:
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/frontyard_sub?video=copy&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - detect 
        - path: rtsp://127.0.0.1:8554/frontyard_main?video=copy&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - record
            
    detect:
      fps: 5
      enabled: True
    record:
      enabled: True
      retain:
        days: 1
        mode: motion
      events:
        retain:
          default: 1
    snapshots: # <----- Enable snapshots
      enabled: True
      bounding_box: true
      retain:
        default: 15
    objects:
      track:
      - person
      - dog
      - cat
      - bird
    live:
      stream_name: frontyard_main
  backdeck:
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/backdeck_sub?video=copy&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - detect

        - path: rtsp://127.0.0.1:8554/backdeck_main?video=copy&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - record

    detect:
      fps: 5
      enabled: True
    record:
      enabled: True
      retain:
        days: 1
        mode: motion
      events:
        retain:
          default: 1
    snapshots: # <----- Enable snapshots
      enabled: True
      bounding_box: true
      retain:
        default: 15
    objects:
      track:
      - person
      - dog
      - cat
      - bird
    live:
      stream_name: backdeck_main
  pool:
    ffmpeg:
      inputs:

        - path: rtsp://127.0.0.1:8554/pool_main?video=copy&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - record
            
        - path: rtsp://127.0.0.1:8554/pool_sub?video=copy
          input_args: preset-rtsp-restream
          roles:
            - detect 
    detect:
      fps: 5
      enabled: True
    record:
      enabled: True
      retain:
        days: 1
        mode: motion
      events:
        retain:
          default: 1
    snapshots: # <----- Enable snapshots
      enabled: True
      bounding_box: true
      retain:
        default: 15
    objects:
      track:
      - person
      - dog
      - cat
      - bird
    live:
      stream_name: pool_main
```

### Relevant log output

```shell
2024-02-08 03:48:13.369608562  [INFO] Starting go2rtc healthcheck service...
2024-02-08 03:48:18.373232209  2024/02/08 14:48:18 [error] 148#148: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /api/version HTTP/1.1", upstream: "http://127.0.0.1:5001version", host: "127.0.0.1:5000"
2024-02-08 03:48:25.434483170  [2024-02-08 14:48:25] peewee_migrate.logs            INFO    : Starting migrations
2024-02-08 03:48:25.436839852  [2024-02-08 14:48:25] peewee_migrate.logs            INFO    : There is nothing to migrate
2024-02-08 03:48:25.440167004  [2024-02-08 14:48:25] frigate.app                    INFO    : Recording process started: 839
2024-02-08 03:48:25.441447132  [2024-02-08 14:48:25] frigate.app                    INFO    : go2rtc process pid: 88
2024-02-08 03:48:25.454912639  [2024-02-08 14:48:25] detector.coral                 INFO    : Starting detection process: 849
2024-02-08 03:48:25.458257674  Process detector:coral:
2024-02-08 03:48:25.458341779  [2024-02-08 14:48:25] frigate.app                    INFO    : Output process started: 851
2024-02-08 03:48:25.458452558  [2024-02-08 14:48:25] frigate.detectors.plugins.edgetpu_tfl INFO    : Attempting to load TPU as usb
2024-02-08 03:48:25.458540703  [2024-02-08 14:48:25] frigate.detectors.plugins.edgetpu_tfl ERROR   : No EdgeTPU was detected. If you do not have a Coral device yet, you must configure CPU detectors.
2024-02-08 03:48:25.459334506  Traceback (most recent call last):
2024-02-08 03:48:25.459345269    File "/usr/lib/python3/dist-packages/tflite_runtime/interpreter.py", line 160, in load_delegate
2024-02-08 03:48:25.459346619      delegate = Delegate(library, options)
2024-02-08 03:48:25.459351142    File "/usr/lib/python3/dist-packages/tflite_runtime/interpreter.py", line 119, in __init__
2024-02-08 03:48:25.459352259      raise ValueError(capture.message)
2024-02-08 03:48:25.459355854  ValueError
2024-02-08 03:48:25.459368552  
2024-02-08 03:48:25.459369649  During handling of the above exception, another exception occurred:
2024-02-08 03:48:25.459370357  
2024-02-08 03:48:25.459371274  Traceback (most recent call last):
2024-02-08 03:48:25.459386721    File "/usr/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
2024-02-08 03:48:25.459387647      self.run()
2024-02-08 03:48:25.459388757    File "/usr/lib/python3.9/multiprocessing/process.py", line 108, in run
2024-02-08 03:48:25.459389755      self._target(*self._args, **self._kwargs)
2024-02-08 03:48:25.459397377    File "/opt/frigate/frigate/object_detection.py", line 102, in run_detector
2024-02-08 03:48:25.459398443      object_detector = LocalObjectDetector(detector_config=detector_config)
2024-02-08 03:48:25.459399559    File "/opt/frigate/frigate/object_detection.py", line 53, in __init__
2024-02-08 03:48:25.459400514      self.detect_api = create_detector(detector_config)
2024-02-08 03:48:25.459401751    File "/opt/frigate/frigate/detectors/__init__.py", line 18, in create_detector
2024-02-08 03:48:25.459422261      return api(detector_config)
2024-02-08 03:48:25.459423496    File "/opt/frigate/frigate/detectors/plugins/edgetpu_tfl.py", line 41, in __init__
2024-02-08 03:48:25.459425281      edge_tpu_delegate = load_delegate("libedgetpu.so.1.0", device_config)
2024-02-08 03:48:25.459426457    File "/usr/lib/python3/dist-packages/tflite_runtime/interpreter.py", line 162, in load_delegate
2024-02-08 03:48:25.459451937      raise ValueError('Failed to load delegate from {}\n{}'.format(
2024-02-08 03:48:25.459459890  ValueError: Failed to load delegate from libedgetpu.so.1.0
```

### FFprobe output from your camera

```shell
Frigate will not start
```

### Frigate stats

```json
Frigate will not start
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

### Network connection

Wired

### Camera make and model

Reolink 

### Any other information that may be helpful

Updated from HAOS 10.4 to 10.5 Frigate would not start after reboot

**Comment by NickM-27** [comment]
The coral is not detected. Do you see the coral in the hardware page in HAOS settings

**Comment by Ralphy140** [comment]
From my research it does not look like the USB Coral show up in the hardware list for HA, The Coral was working fine before the upgrade and no physical changes. I did see the error about the coral so removed it from the config with the same traceback and error accessing port 5001

**Comment by rysciguy** [comment]
Restarting my machine -- not just the virtual machine running HAOS with the Frigate add-on, but the actual machine running Proxmox -- solved this issue for me. I can also confirm that my Coral device (PCIE, not USB) does not appear in the Hardware page in settings.

**Comment by rysciguy** [comment]
> Restarting my machine -- not just the virtual machine running HAOS with the Frigate add-on, but the actual machine running Proxmox -- solved this issue for me. I can also confirm that my Coral device (PCIE, not USB) does not appear in the Hardware page in settings.

Update: Frigate crashed within 12 hours and would not start back up. Restarting the add-on gave the same `No EdgeTPU was detected. If you do not have a Coral device yet, you must configure CPU detectors.` error as before. When I rebooted the HAOS VM, I received this failure message in Proxmox:

``kvm: ../hw/pci/pci.c:1562: pci_irq_handler: Assertion `0 <= irq_num && irq_num < PCI_NUM_PINS' failed.
TASK ERROR: start failed: QEMU exited with code 1``

To confirm the issue was caused by the Coral, I turned off hardware passthrough for the PCIE device in Proxmox (which I imagine has the same effect as demating the connector from the VM's perspective), and rebooted the VM again, which succeeded this time. But turning PCIE passthrough back on caused another failed start.

**Once again the solution was to reboot the host machine.** This fixed my issue for now, but it may not be a permanent solution. Hopefully this info is helpful for others experiencing similar issues, and I'm not just hijacking the thread with irrelevant information!
